components/wmbus/sensor/__init__.py

```diff
@@ -39,6 +39,9 @@
 CONF_MODE = "mode"
 
 UNIT_LITER = "l"
+UNIT_LITER_METER_PER_HOUR = "l/h"
+UNIT_GIGA_JOULE = "GJ"
+UNIT_DAY = "d"
```

Comment:
Maybe "hours" will be better. Just like telegram from sharky.

Reply:
No
Days is better 9 892 h vs 412 days ?